src/battle-log.ts

```diff
@@ -938,7 +950,7 @@ class BattleLog {
 		// allows T, however it's more practical to also allow spaces.
 		return sanitized.replace(
 			/<time>\s*([+-]?\d{4,}-\d{2}-\d{2})[T ](\d{2}:\d{2}(?::\d{2}(?:\.\d{3})?)?)(Z|[+-]\d{2}:\d{2})?\s*<\/time>/ig,
-		this.localizeTime);
+		this.localizeTime).replace(/<(.*)&amp;(.*)>/ig, '<$1&$2>');
```

Comment:
```suggestion
		this.localizeTime);
```

:[ This will break the HTML sanitizer and create invalid output. You'll need a different solution to this problem.

Reply:
I'm pretty sure you don't need this, anyway. `<iframe src="&amp;">` does in fact create an iframe with a source of `&`, if you were worried about that.

Reply:
hm, alright